queries.sql

```diff
@@ -8,4 +8,40 @@ select date_of_birth from animals where name = 'Agumon' OR name = 'Pikachu';
 select name, escape_attempts from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name != 'Gabumon';
-select * from animals where weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+select * from animals where weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT name, species FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' where name like '%mon';
+UPDATE animals SET species = 'pokemon' where species IS NULL;
+
+BEGIN;
```

Comment:
- [ ] Before this query, you are missing the query below, kindly implement it. 
<img width="966" alt="Screenshot 2022-07-21 at 17 27 23" src="https://user-images.githubusercontent.com/64774936/180267424-f2000234-b254-47f2-bd92-3fa96f2e0c5d.png">

- It would be something like this
   ``` BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

Reply:
DONE

Reply:
This issue is well resolved! ✔️